jcl/src/java.base/share/classes/com/ibm/oti/util/Util.java

```diff
@@ -286,23 +298,43 @@ public static void appendLnTo(Appendable buf) {
  * @param e StackTraceElement to add
  * @param elementSource location of the source file.  May be null.
  * @param buf Receiver for the text
- * @param extended add additional information
+ * @param includeExtendedInfo Whether or not to include extended info, such as classloader name or module version
  */
-public static void printStackTraceElement(StackTraceElement e, Object elementSource, Appendable buf, boolean extended) {
+public static void printStackTraceElement(StackTraceElement e, Object elementSource, Appendable buf, boolean includeExtendedInfo) {
 	/*[IF Sidecar19-SE]*/
```

Comment:
This could be changed to JAVA_SPEC_VERSION >= 11, and the comment in the ENDIF to match.